_posts/2023-11-23-geospatial-techno.md

```diff
@@ -0,0 +1,81 @@
+
+#### Uninstallation
+GeoServer can be uninstalled in two ways:
+- By running the uninstall.exe file in the directory where GeoServer was installed
+- By standard Windows program removal
+
+### Web Archive
+GeoServer is packaged as a web-archive (WAR) for use with an application server such as Apache Tomcat or Jetty. It has been mostly tested using Tomcat, and so is the recommended application server. There are reasons for installing it such as it is widely used, well-documented, and relatively simple to configure. GeoServer requires a newer version of Tomcat (7.0.65 or later) that implements Servlet 3 and annotation processing. Other application servers have been known to work, but are not guaranteed.
+
+#### Installation
+- Make sure you have a JRE installed on your system, then download Apache Tomcat from its website(https://tomcat.apache.org). For the Windows installation package, scroll down and choose the 32bit/64bit Windows Service Installer option.
+- Configure Tomcat by selecting components, setting up a username and password, and specifying memory settings. So, before start the Tomcat service, you have to configure the memory settings that will use for Java VM. To do it, open the Tomcat9w from the bin folder, then click on the Java tab. This tab allows for configuration of memory settings, including initial and maximum memory pool sizes. Recommended values are 512MB for the initial memory pool and 1024MB for the maximum memory pool.
+- Start Tomcat service and verify its functionality, then navigate to localhost:8080, and get the Tomcat9 web page.
+- Navigate to the GeoServer.org and Download page. Select Web Archive on the download page from the version of GeoServer that you wish to download.
+- Deploy the GeoServer web archive as you would normally. Often, all that is necessary is to copy the GeoServer.war file to the Tomcat’s webapps directory, then the application will be deployed automatically.
+- Now to access the Web administration interface, open a browser and navigate to localhost:8080 and press Manager App button. Enter the username and password of apache tomcat. Click on the start button for the GeoServer. Once it has started, click the GeoServer link. This will take you to the GeoServer web page.
+
+#### Uninstallation
+Stop the container application. Remove the GeoServer webapp from the container application’s webapps directory. This will usually include the GeoServer.war file as well as a GeoServer directory.
+
+### Difference between GEOSERVER.war and GEOSERVER.exe?
+- GeoServer.exe installs GeoServer as a windows service running inside Jetty. GeoServer.war is a platform independant package that needs to be installed into a container server e.g. tomcat, jboss, …
+The downside of using the Windows Installer is that you have to be on a Windows machine, you also get less choice of servlet container.
```

Comment:
This sentence does not make sense to me? This page is dedicated to installing on windows so being on a windows machine is not a downside?

```
- Using the `geoserver.exe` installer is a reliable way to setup GeoServer as a windows background service. The downside is the included Jetty application server is managed using text files (jetty.ini) once installed.
```